javascript/javascript1/week4/voiceReconderComandsApp.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+
+function getReply(command) {
+        let names = [];
+        let todo = [];
+
+        if (command.startsWith("Hello ")) {
+          function getName() {
+            let name = command.split(" ").slice(-1);
+            names.push(name);
+            console.log("Nice to meet you " + name);
+          }
+          getName();
+        }
```

Comment:
Good use of functions however for this particular case, I think it adds an unnecessary level of complexion. Since you're defining the funciton and using it in the same statement, it would be cleaner to just provide the logic directly in the statement! 

Same for below as well!